ogr/services/github.py

```diff
@@ -55,6 +55,10 @@ def __init__(
         self.github = self.github_class(login_or_token=self._token)
         self.read_only = read_only
 
+    def __str__(self)-> str:
+        return (
+            f"Github-Service"
```

Comment:
Please, use the same names as in code:
```suggestion
            f"GithubService"
```
It is relevant for other methods as well... For classes as well for attributes.

-----

What about adding `read_only` attribute. (You can add it also to the `PagureService`.)